sc-platform-api/src/api/user/user.controller.ts

```diff
@@ -1,4 +1,32 @@
-import { Controller } from '@nestjs/common';
+import { Controller, Get, Param, UseGuards } from '@nestjs/common';
+import { ApiTags } from '@nestjs/swagger';
+import { ApiEndpoint } from 'src/documentation/decorators';
+import { UserService } from './user.service';
+import { UserMapper } from './user.mapper';
+import { UserDocumentation } from 'src/documentation/user';
+import { UserByIdPipe } from './user.pipe';
+import { AccessGuard } from 'src/security/jwt/access/access.guard';
 
-@Controller('user')
-export class UserController {}
+@ApiTags('User')
+@Controller({
+  path: '/users',
+})
+export class UserController {
+  constructor (
+    private userService: UserService,
+    private userMapper: UserMapper,
+  ) {}
+
+  @ApiEndpoint({
+    summary: 'Return user\'s data',
+    documentation: UserDocumentation.GET_USER,
+  })
+  @UseGuards(AccessGuard)
```

Comment:
You can do it without `@UseGuards` decorator
```suggestion
  @ApiEndpoint({
    summary: 'Return user\'s data',
    guards: AccessGuard,
    documentation: UserDocumentation.GET_USER,
  })
```